pillar/base/tls.sls

```diff
@@ -61,3 +61,77 @@ tls:
     svn.psf.io:
       roles:
         - hg
+
+  acme_certs:
+    bugs.python.org:
```

Comment:
I don't see a reason to have a separate instance here just for bugs.python.org. The same cert can be used for the load balancer and the roundup box.